### DISCUSSION TITLE
Exit - links back to main page on Exit

### DISCUSSION THREAD

**Discussion by ozdotcom** [CarlosPietrasanta/Tafe230130#7] [Q&A] [answered]
exitButton_Click event.

@Jasonlovedataanalyst 
@CarlosPietrasanta 

Are we using the Universal Calculator page link for the exitButton_Click events to exit from the Mortgage Calculator, Currency Calculator, Trip Calculator.

At this stage I have used this, exits application.

Application.Current.Exit();

Thoughts

Sharryn

**Comment by ozdotcom** [comment, marked as answer]
Found the answer in session 6 PowerPoint. 

![image](https://user-images.githubusercontent.com/87345204/227387683-58713218-a3c7-48fc-963d-6b761802e9be.png)

**Reply by Jasonlovedataanalyst** [reply]
I am sorry that I am struggling with the xaml file now , and will get back to you , when I touch the cs part

**Comment by CarlosPietrasanta** [comment]
Sorry for the late reply guys.

Yes, I used the same you found Sharryn, to go back to the main menu instead of exiting the app. I think it makes more sense than entering a calculator and only being able to exit completely.

Jason, let us know what is being an issue for you and we'll be more than happy to assist.
